### PR TITLE
Issue #115 gome harvester stuck

### PR DESCRIPTION
**Pull request by davidcordeiro-deimos**
Fixes #

### Proposed fixes:
Regarding issue #115, the solution implemented is to make this time interval configurable through the harvest configuration. An example of a harvester configuration is:

```
{
   "start_date": "2019-07-01",
   "time_interval": 20,
    "make_private": false
}
```

### Features:

- Harvester log

- [X] includes new  features
- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [X] includes bugfix

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by davidcordeiro-deimos** [comment]
As an additional note, if "time_interval" is not specified in the harvester configuration, the default value is 15.

**Comment by davidcordeiro-deimos** [comment]
@georgiana-b I do not know how familiar you are with the intricacies of the harvesters and their jobs. However, due to this harvester being one of the first to be created at the beginning of the project, it lacks most of the current robustness mechanisms.

One of the said mechanisms is the ability to handle datasets that already have been harvested by giving it a status of unchanged when sent to the fetch stage. However, as @joaandrade mentioned in #115, for now, we should avoid spending more effort on this harvester.

> In this harvester, the start_date always remains the same. Also get_last_harvesting_date() returns None even though there are previously harvested datasets for this source. Could you check why that is?

The function get_last_harvesting_date() returns the date of the last product collected **by that harvester**, it does not check products collected from the same source by another harvester (this limitation is valid for all harvesters)
  
So due to both limitations mentioned previously if you setup the harvester to when all the data has been collected or there is no data (i.e. within the start data + time_interval chosen) the function get_last_harvesting_date() will keep returning None, and this is the expected behavior assuming the previous limitations.

In conclusion, if within the harvester configuration you set up the initial date as the last product collected (or if said harvester was the one to collect the most recent product, you can leave this field as it is) and the time_interval big enough to overcome the largest product drought, I do not foresee any scenario in which the harvester will get stuck. By looking into the regularity of products in the last few years, the largest drought was in 2019 of 11 days, thus I would say that 30 is enough should be enough, while not creating jobs that run for long periods of time.

**Comment by georgiana-b** [comment]
According to @davidcordeiro-deimos (https://github.com/NextGeoss/ckanext-nextgeossharvest/issues/115#issuecomment-571202999), the harvester got stuck on `2019-07-31` because it was looking for new datasets only in the next 10 days and there were no datasets at the source in next 10 days.

 Therefore the solution proposed in this PR is to make this interval configurable to a higher number so that the harvester doesn't get blocked into those data gaps. When we deploy we will have to create a new harvester with a configuration like the following:

```
{
   "start_date": "2019-08-01",
   "time_interval": 30,
   "make_private": false
}
```

⚠️ Very important to use `2019-08-01` as a start date. If we start from July `2019-07-01` the harvester will most probably get stuck again. That's because the harvester cannot recognize and move over previously collected datasets.
We will keep running into this issue whenever we try to run another Gome2 harvester on a previously harvested timeframe. The second harvester will remain stuck. 

Otherwise, if we have only one harvester that runs continuously, it keeps moving forward and collecting datasets as expected. 

I will not insist further since everyone else seems to think this harvester is not worth the effort.

@JGulic Please share your thoughts. If you think the behaviour described above is enough for our needs at the moment, feel free to merge the PR.

**Comment by JGulic** [comment]
I took a look at this issue and I've tested the PR locally.

Basically the issue remains even after we apply the PR. If we take into consideration that we are missing data for 50 days and the time_interval is set to 20, the harvester is again going to be stuck.
We can merge the PR and start the harvester with a time_interval set to a higher number, but I want to highlight here that **the problem is not solved with this, it may well appear in the future**.

_**Since the decision is not to spend more time on fixing this harvester, we are going to merge the PR and close the issue, and we should all have in mind that this issue may appear in the future again.**_

**Comment by joaandrade** [comment]
Hi @JGulic, you're right. In any case, the operational team (Deimos Space) will be aware of the problem for this harvester.
@davidcordeiro-deimos did you update the Wiki with the new configuration variable for this harvester?

**Comment by JGulic** [comment]
@joaandrade since this PR has already been merged, Please open a new issue and a new PR for adding the changes that we need in the configuration.
Thank you
